doc/source/how-to/releasing.rst

```diff
@@ -212,39 +212,134 @@ script. If you are operating out of a GitHub CI pipeline, email the PyAnsys Core
 team at `pyansys.core@ansys.com <mailto:pyansys.core@ansys.com>`_ for the
 required ``PAT`` user name and ``PYANSYS_PYPI_PRIVATE_PAT`` password.
 
-Assuming that you are already in a Python repository's ``dist/`` directory that
-contains your wheel, source distribution, or both, you can upload to the private
-repository with:
+For uploading packages, `Twine <https://pypi.org/project/twine/>`_ is required.
+Install this tool by running:
 
-.. code-block:: bash
+.. code-block:: text
+
+    python -m pip install twine
+
+Uploading to private PyPI using GitHub actions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A good practice is to publish a new release of your project to both public and
```

Comment:
@jorgepiloto I think it is not necessary to publish to both if your project is public. Maybe the first time for testing but after that it should be ok. What do you think?

Reply:
I was thinking to have synced versions both in public and private PyPI. Should we encourage developers to stop publishing to private PyPI once the project is public?

Reply:
I would say yes. It would be confusing to have it both in two different locations.
The private repo was created for people who cannot expose publicly their work yet.

Reply:
Just deleted this line in https://github.com/pyansys/dev-guide/pull/153/commits/a02dc19a97440cc1249031627640f376cd5691f8.

Reply:
Yep I agree. Once the project goes public, we should only keep the public PyPi release :) We could even think of "deleting" it from the private PyPi so that people don't try to still download it from there. What do you think @jorgepiloto @MaxJPRey?

Reply:
I would say yes to clean the repo as much as we can.

Reply:
I agree. Adding this to the content 👍🏽